docs/visualization/plotting-1d-data.ipynb

```diff
@@ -464,6 +478,24 @@
    "source": [
     "plot(d4, filename='my_1d_figure.pdf')"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Help\n",
+    "\n",
+    "Inspect help to view all options for plot"
```

Comment:
I think this is expected of any well contructed python package, I don't think we need to spell it out in the docs